builder/Dockerfile

```diff
@@ -40,3 +39,8 @@ RUN sudo apt-get update &&  apt-get install yarn
 RUN apt-get autoremove -y --purge \
     && apt-get clean \
     && rm -Rf /tmp/*
+
+RUN useradd --shell /bin/bash -u 1000 -o -c "" -m dev
```

Comment:
@mathieu-ducrot I found I have that part uncommited so maybe this is why you add right conflict

Reply:
@nicolas-bastien i did not succed to reproduce the permission conflict that i had, but it's still ok to add this.